packages/debugger-extension/schema/main.json

```diff
@@ -40,7 +40,7 @@
       "selector": "body"
     },
     {
-      "command": "debugger:continue",
+      "command": "debugger:pause-continue",
```

Comment:
```suggestion
      "command": "debugger:continue",
```

This is highlighted by the failing test.

Reply:
Thanks @fcollonval